tools/kubectl-ownerref/main.go

```diff
@@ -5,57 +5,188 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
 	"strings"
 
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/printers"
 	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/cmd/get"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/utils/pointer"
 
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/client-go/util/homedir"
-)
-
-var (
-	ns = flag.String("ns", "", "namespace")
+	// Load all auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
 func main() {
-	var kubeconfig *string
-	if homeDir := homedir.HomeDir(); homeDir != "" {
-		kubeconfig = flag.String("kubeconfig", filepath.Join(homeDir, ".kube", "config"), "kubeconfig file")
-	} else {
-		kubeconfig = flag.String("kubeconfig", "", "kubeconfig file")
+	rootCommand := cobra.Command{}
```

Comment:
```suggestion
	rootCommand := cobra.Command{}
	rootCommand.CompletionOptions.DisableDefaultCmd = true
```
`completion` sub-command exist on current code.
Is it explicitily? If not, disable `completion` sub-command by this.

Reply:
Applied - thank you!